css/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0 auto;
+}
+
+.navbar {
+  width: 1080px;
+  height: 150px;
+  border: 5px dotted blue;
+  float: left;
+  clear: right;
```

Comment:
Instead of adding clear to each of your tags, create a CSS rule for a clearfix class with the property "clear: both".  In the HTML file add an empty div with a class of clearfix.

This will keep your html / css focused on the content while letting the clearfix divs clean up after floats.